src/web/src/pages/_app.tsx

```diff
@@ -27,7 +27,7 @@ const App = ({ Component, pageProps }: AppProps) => {
     if (jssStyles) {
       jssStyles.parentElement?.removeChild(jssStyles);
     }
-  }, []);
```

Comment:
Why remove this?  Won't this mean it runs on every (re)render?

Reply:
Yeah, I deleted by accident. Fixed now!